src/parser.cpp

```diff
@@ -440,6 +440,10 @@ const wchar_t *parser_t::is_function() const {
     return result;
 }
 
+const wchar_t *parser_t::get_function() {
```

Comment:
I don't like needlessly long symbol names but `get_function_name()` or `get_func_name()` better describes the purpose of the function.